packages/vscode-extension/src/extension.ts

```diff
@@ -193,6 +193,16 @@ export async function activate(context: vscode.ExtensionContext) {
   registerTeamsfxTaskAndDebugEvents();
 
   await handlers.cmdHdlLoadTreeView(context);
+
+  // Register local debug button
+  const selectDebugCmd = vscode.commands.registerCommand(
+    "fx-extension.selectAndDebug",
+    handlers.selectAndDebugHandler
+  );
+  context.subscriptions.push(selectDebugCmd);
+
+  vscode.commands.executeCommand("setContext", "fx-extension.active", true);
```

Comment:
Will this context cleared when deactivate?

Reply:
Now the status will be changed if the window is reloaded.